x-pack/legacy/plugins/infra/public/components/metrics/sections/series_chart.tsx

```diff
@@ -46,7 +46,7 @@ export const AreaChart = ({ id, color, series, name, type, stack }: Props) => {
       visible: true,
     },
     point: {
-      visible: true,
+      visible: false,
```

Comment:
Do you think it might be easy to over look the `visible: false` part? If we just removed the entire `point` object, would that make the point disappear also?